ddtrace/debugging/_debugger.py

```diff
@@ -283,6 +288,22 @@ def __init__(self, tracer: Optional[Tracer] = None) -> None:
 
         log.debug("%s initialized (service name: %s)", self.__class__.__name__, service_name)
 
+    def _install_file_probes(self, path: str) -> None:
```

Comment:
Should we maybe unify this with the `di_config.probe_file` check above into, e.g.
```suggestion
    def _load_local_config(self) -> None:
```
and call this in the `__init__`?